src/Core/Addon/Theme/Theme.php

```diff
@@ -25,8 +25,19 @@
  */
 namespace PrestaShop\PrestaShop\Core\Addon\Theme;
 
-abstract class AbstractTheme implements AddonInterface
+use PrestaShop\PrestaShop\Core\Addon\AddonInterface;
+
+class Theme implements AddonInterface
 {
+    public function __construct(array $attributes)
+    {
+        foreach ($attributes as $attr => $value) {
+            $this->{$attr} = $value;
```

Comment:
I think this fails if, for instance, `$attributes` has a `onInstall` key. Might happen since `$attributes` comes from a "user"-defined file.